matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/crypto/crosssigning/CrossSigningService.kt

```diff
@@ -64,5 +64,5 @@ interface CrossSigningService {
                          otherDeviceId: String,
                          locallyTrusted: Boolean?): DeviceTrustResult
 
-    fun getTrustLevelForUsers(userIds: List<String>): RoomEncryptionTrustLevel
+    suspend fun getTrustLevelForUsers(userIds: List<String>): RoomEncryptionTrustLevel
```

Comment:
This is not used anymore. Remove? (Also for the moment we do not expose suspendable fun in the API)

Reply:
As not used we can remove for now. (i thought I heard we will soon expose suspendable )

Reply:
ok